sklearn/pipeline.py

```diff
@@ -867,8 +872,11 @@ def _validate_transformers(self):
             if (not (hasattr(t, "fit") or hasattr(t, "fit_transform")) or not
                     hasattr(t, "transform")):
                 raise TypeError("All estimators should implement fit and "
-                                "transform. '%s' (type %s) doesn't" %
-                                (t, type(t)))
+                                "transform. '%s' (type %s) doesn't"
+                                % (t, type(t)))
```

Comment:
You can put the `%` on the previous line so that it does not change the existing code.

Reply:
That would amount to more than 79 characters in that line, wouldn't it?

Reply:
Sorry, it's not something that can easily be seen on GitHub only. Alright then!

Reply:
Oh, I misunderstood you, you're right. I changed it back to the previous version